templates/widget/contribution.ts

```diff
@@ -0,0 +1,70 @@
+import { injectable } from 'inversify';
```

Comment:
We don't add the file headers for the extension, it's more up to the developer using the extension themselves to provide one (since in the end it is their extension not ours).